Chaincase.UI/Services/Extensions.cs

```diff
@@ -26,7 +26,7 @@ public static IServiceCollection AddUIServices(this IServiceCollection services)
 			services.AddTransient<LoadWalletViewModel>();
 			services.AddTransient<WalletInfoViewModel>();
 			services.AddTransient<NewPasswordViewModel>();
-			services.AddTransient<CoinJoinViewModel>();
+			services.AddSingleton<CoinJoinViewModel>();
```

Comment:
I feel like this should be scoped however I'm not sure how to control the scope of a VM injected in a Blazor page @Kukks  do you know what the scope ought to be? It would be great to maintain it up to a parent but no further. Probably best to keep the memory footprint as small as possible

Reply:
@armins88 you asked me once about the memory of pages & viewModels. Here's a peek into that